src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -65,6 +66,20 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
         self.STR_DPKG_WAS_INTERRUPTED = "E: dpkg was interrupted, you must manually run 'sudo dpkg --configure -a' to correct the problem."
         self.ESM_MARKER = "The following packages could receive security updates with UA Infra: ESM service enabled:"
 
+        # Ubuntu Pro Client pre-requisite checks.
+        try:
+            self.__pro_client_prereq_met = False
+            self.ubuntu_pro_client = UbuntuProClient(env_layer, composite_logger)
+
+            # Add condition to check if python3 is also installed.
```

Comment:
Is this a TODO? Prefix with TODO: so both reviewers and you know it's a todo vs. a descriptive comment.

Reply:
This is a TODO. Will update this in next iteration